tests/Features/Expect/sequence.php

```diff
@@ -40,7 +42,7 @@ function ($expectation) {
     expect(static::getCount())->toBe(16);
 });
 
-test('fails if the number of iterable items is greater than the number of expectations', function () {
```

Comment:
Why this test changed?

Reply:
there is a small mistake in the test description: when using `sequence()` we are allowed to have less callbacks than the number of iterable items.

we can even have just one callback which is called for every item.

we cannot - however - have less items than the callbacks, in that case it should fail.

the `sequence()` expectation is already behaving that way, there is just a little typo in the test description that is misleading 👍 

also, we are throwing `OutOfRangeException` instead of running `Assert::assertLessThanOrEqual()` because the latter shows a confusing error and runs an assertion that the end-user is not explicitly asking for. on the other hand the thrown exception is exhaustive and tells the user why the expectation failed 👍